freqtrade/strategy/default_strategy.py

```diff
@@ -13,6 +13,7 @@ class DefaultStrategy(IStrategy):
     Default Strategy provided by freqtrade bot.
     You can override it with your own strategy
     """
+    strategy_version: int = 2
```

Comment:
Well, I think this should be simplified for a user, so no `: int`, then it should be like a constant, so in upper letters, then it's the version of the interface, not the strategy itself, I propose
```
STRATEGY_INTERFACE_VERSION = 2
```
or
```
STRATEGY_INTERFACE_VER = 2
```
or
```
strategy_interface_version = 2
```
if you mind this to be in capital letters.

We also could drop `strategy` from the name since it's in the Strategy class, so
`INTERFACE_VERSION` or `interface_version` can be used as well.

Reply:
`INTERFACE_VERSION` seems the best (short, and as a constant)